ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -11,23 +10,19 @@
       <name>Stéphane Marchand</name>
       <email>stephane.marchand@ens-lyon.fr</email>
     </author>
-    <contributor>
-      <name>Isabelle Colon de Carjaval</name>
-      <email>isabelle.colondecarvajal@ens-lyon.fr</email>
-    </contributor>
+    <author>
+      <name>Anatole Lucet</name>
+      <email>anatole.lucet@ens-lyon.fr</email>
+    </author>
```

Comment:
yes, she was supposed to get involved in this project, but she finally did not participate.